utils/log-collector

```diff
@@ -16,7 +16,7 @@ set -euo pipefail
 
 usage() {
 cat << EOF
-Usage: ./bk-log-collector -s <stack-name> -i <instance-id>
+Usage: ./log-collector -s <stack-name> -i <instance-id>
```

Comment:
Could this use `$0` or something like that so that it's dynamic and always correct?

Reply:
good suggestion!